content/articles/domains-uk.markdown

```diff
@@ -61,11 +61,66 @@ Instead of creating a new contact in DNSimple, it's perfectly fine to reuse an e
 
 ## Transferring a .UK domain to DNSimple {#transfer}
 
-To transfer a UK domain (i.e. example.uk or example.co.uk) into DNSimple, follow [these instructions](/articles/transferring-uk-domain).
+`.UK` and `.CO.UK` domains have a different transfer process than most other domain registrations. These steps explains some of those differences.
+
+### Before you start
+
+<warning>
+It is important you follow these steps in strict order, other case, there is a risk of loosing your domains registration.
```

Comment:
I'd change this to:

"It is important you follow these steps in strict order. Otherwise, there is a risk of losing your domain's registration."